detect_secrets/plugins/jwt.py

```diff
@@ -2,8 +2,11 @@
 This plugin finds JWT tokens
 """
 import base64
+import jwt
```

Comment:
Historically, I've attempted to reduce the number of dependencies that `detect-secrets` needs to install. This is probably the reason why `is_formally_valid` doesn't use the library. Furthermore, if we require this, then it would require all users to install `pyjwt`, even if they don't use this plugin.

If we're not verifying signatures, this should be an easier fix: please try to do this manually, rather than use a library. The good news is that you can use a lot of the same logic as already defined in `is_formally_valid`.